flight/mcu/lib/teensyShared/telemetry.cpp

```diff
@@ -16,17 +16,18 @@
 // #define SERIAL_PLOTTING
 
 #include <telemetry.h>
-
 #include <limits>
 
-template <typename T>
-T inv_convert_range(float val, float range) {
-    size_t numeric_range = (int64_t)std::numeric_limits<T>::max() - (int64_t)std::numeric_limits<T>::min() + 1;
-    float converted = val * (float)numeric_range / range;
-    return std::max(std::min((float)std::numeric_limits<T>::max(), converted), (float)std::numeric_limits<T>::min());
+Telemetry tlm;
+
+template<typename T>
+T inv_convert_range(float val, float range){
```

Comment:
Definitely some more documentation as to what this does. It's clear that it's mean to convert ranges before sending to ground, but that isn't apparent if someone isn't already familiar with our telemetry system.

Reply:
@NPIPHI Honestly, I forget how this function works every time I stop looking at it, so could you write the docs for this one?

Reply:
yeah lol